explanations/duplicate-check-inputs.md

```diff
@@ -0,0 +1,3 @@
+In Python package derivations, `propagatedBuildInputs` is used to list depedencies that are required to be available at run time, in addition to build time. `checkInputs` lists dependencies that are required specifically during the `checkPhase`.
```

Comment:
I don't want to nit here but most of the time they are required at runtime. Build only dependency are in build or native build inputs.